packages/animations/browser/src/dsl/animation_ast_builder.ts

```diff
@@ -311,6 +319,16 @@ export class AnimationAstBuilderVisitor implements AnimationDslVisitor {
           return;
         }
 
+        if (this._driver.validateAnimatableStyleProperty) {
```

Comment:
If this is a dev-mode-only logic, I'd propose adding `ngDevMode` here as well:
```suggestion
        if (ngDevMode && this._driver.validateAnimatableStyleProperty) {
```